### PR TITLE
Automated cherry pick of #68920: Enable insertId generation, update Stackdriver Logging Agent

### PR DESCRIPTION
**Pull request by qingling128**
Cherry pick of #68920 on release-1.11.

#68920: Enable insertId generation, update Stackdriver Logging Agent

### CODE REVIEW

**Comment by qingling128** [comment]
Could someone help with the `/ok-to-test` tag? @kawych @bowei  @x13n

**Comment by x13n** [comment]
/lgtm cancel
Actually, after talking with @MaciekPytel I think we should have wildcard tolerations for per-node agents (metadata agent & fluentd).

**Comment by qingling128** [comment]
https://github.com/kubernetes/kubernetes/pull/69813 is created to add tolerations.

Can we unblock this one now (since they are not depending on each other)?

**Comment by x13n** [comment]
I'd wait for #69813 to merge, then merge this (to make sure this change is not patch-released without the other one), so putting on hold. Otherwise looks good.
/lgtm
/approve
/hold

**Comment by qingling128** [comment]
@x13n - Sounds good. I will wait for the other one to merge before merging this one.
@piosz - Mind taking a quick look?

**Comment by foxish** [comment]
ping - approvers

File: cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml
Comment (x13n) [inline cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml]: This actually removes the tolerations now, which it shouldn't. Please rebase & make sure they are not removed.

**Comment by MaciekPytel** [comment]
/approve
/hold 
Please reply to @x13n comment before merging. Feel free to cancel hold once it's resolved.

**Comment by qingling128** [comment]
Rebased. PTAL @x13n @kawych @MaciekPytel

**Comment by qingling128** [comment]
Seems the PR has both "approved" and "do-not-merge/cherry-pick-not-approved" labels, which blocks the merge.

I'm a bit confused. Any idea who I should talk to for getting rid of "do-not-merge/cherry-pick-not-approved"?
@x13n @kawych @MaciekPytel

**Comment by x13n** [comment]
This is up to patch release manager for 1.11 branch - @foxish, can you take a look?

**Comment by x13n** [comment]
This is up to patch release manager for 1.11 branch - @foxish, can you take a look?

**Comment by kawych** [comment]
@qingling128 "approved" is for owner approvals (depending on the code location) and "cherrypick-approved" is for branch manager approval. For 1.11 you need approval from @foxish 
/cc @foxish

**Comment by kawych** [comment]
@qingling128 "approved" is for owner approvals (depending on the code location) and "cherrypick-approved" is for branch manager approval. For 1.11 you need approval from @foxish 
/cc @foxish

**Comment by kawych** [comment]
@qingling128 "approved" is for owner approvals (depending on the code location) and "cherrypick-approved" is for branch manager approval. For 1.11 you need approval from @foxish 
/cc @foxish

**Comment by kawych** [comment]
@qingling128 "approved" is for owner approvals (depending on the code location) and "cherrypick-approved" is for branch manager approval. For 1.11 you need approval from @foxish 
/cc @foxish

**Comment by qingling128** [comment]
That makes sense. Thanks for the info!

@foxish - Seems https://github.com/kubernetes/kubernetes/pull/69701 has been approved. Could you take a quick look of this one as well? It should be pretty similar, just to a different release branch.

**Comment by qingling128** [comment]
Friendly ping. @foxish - Do you have a sec to review this one?

**Comment by foxish** [comment]
The branch is frozen for 1.11.4. I'll review this after we cut that today.

**Comment by qingling128** [comment]
Got it. Thanks for the heads up. I'll check back later.

**Comment by qingling128** [comment]
Friendly ping. @foxish - Do you have a sec to review this one?

**Comment by qingling128** [comment]
@foxish - Have you gotten a chance to take a look at this one? Are we still blocked?

**Review by foxish — APPROVED** [review]

**Comment by qingling128** [comment]
Thanks @foxish!

**Pull request merged**
